server/handler/frontend.go

```diff
@@ -55,6 +61,164 @@ func LoadTemplates(c *FilesConfig, basePath string) (templatetree.HTMLTree, erro
+				}
+				if len(predicateInfo.ContributorInfo.Users) > 0 {
+					membershipInfo["Users"] = predicateInfo.ContributorInfo.Users
+				}
+				return membershipInfo
+			case "TargetsBranch", "FromBranch":
+				return map[string][]string{"follows patterns": predicateInfo.BranchInfo.Patterns}
+			case "ChangedFiles", "OnlyChangedFiles":
+				pathInfo := make(map[string][]string)
+				if len(predicateInfo.FileInfo.Paths) > 0 {
+					pathInfo["are in paths"] = predicateInfo.FileInfo.Paths
+				}
+				if len(predicateInfo.FileInfo.IgnorePaths) > 0 {
+					pathInfo["while ignoring paths"] = predicateInfo.FileInfo.IgnorePaths
+				}
+				return pathInfo
+			case "ModifiedLines":
+				modificationInfo := make(map[string][]string)
+				if predicateInfo.FileInfo.AdditionLimit != "" {
+					modificationInfo["number of lines added is "] = []string{predicateInfo.FileInfo.AdditionLimit}
+				}
+				if predicateInfo.FileInfo.DeletionLimit != "" {
+					modificationInfo["number of lines deleted is"] = []string{predicateInfo.FileInfo.DeletionLimit}
+				}
+				if predicateInfo.FileInfo.TotalLimit != "" {
+					modificationInfo["number of total changes is "] = []string{predicateInfo.FileInfo.TotalLimit}
+				}
+				return modificationInfo
+			case "HasLabels":
+				return map[string][]string{"in": predicateInfo.LabelInfo.RequiredLabels}
+			case "HasValidSignaturesBy":
+				membershipInfo := make(map[string][]string)
+				if len(predicateInfo.CommitInfo.Organizations) > 0 {
+					membershipInfo["Organizations"] = predicateInfo.CommitInfo.Organizations
+				}
+				if len(predicateInfo.CommitInfo.Teams) > 0 {
+					membershipInfo["Teams"] = predicateInfo.CommitInfo.Teams
+				}
+				if len(predicateInfo.CommitInfo.Users) > 0 {
+					membershipInfo["Users"] = predicateInfo.CommitInfo.Users
+				}
+				return membershipInfo
+			case "HasValidSignaturesByKeys":
+				return map[string][]string{"Required Keys": predicateInfo.CommitInfo.RequiredKeys}
+			case "Title":
+				patternInfo := make(map[string][]string)
+				if len(predicateInfo.TitleInfo.MatchPatterns) > 0 {
+					patternInfo["matches patterns"] = predicateInfo.TitleInfo.MatchPatterns
+				}
+				if len(predicateInfo.TitleInfo.NotMatchPatterns) > 0 {
+					patternInfo["does not match patterns "] = predicateInfo.TitleInfo.NotMatchPatterns
+				}
+				return patternInfo
+			default:
+				return map[string][]string{}
+			}
+		},
+		"isRequiresNotEmpty": func(result *common.Result) bool {
+			if len(result.Requires.Organizations) != 0 || len(result.Requires.Teams) != 0 || len(result.Requires.Users) != 0 {
+				return true
+			}
+			return false
+		},
+		"getRequires": func(result *common.Result) map[string][]Membership {
+			membershipInfo := make(map[string][]Membership)
+			for _, org := range result.Requires.Organizations {
+				if orgs, ok := membershipInfo["Organizations"]; ok {
+					membershipInfo["Organizations"] = append(orgs, Membership{Name: org, Link: "https://www.github.com/orgs/" + org})
```

Comment:
As mentioned in Slack, the links here need to use the server configuration to set the correct GitHub URL. You can get the base web URL from `config.Github.WebURL`. We'll also need to make sure the generated links behave correctly for both github.com and GitHub Enterprise.